lib/ios/TopBarAppearancePresenter.m

```diff
@@ -32,18 +32,20 @@ - (void)setTransparent:(BOOL)transparent {
 }
 
 - (void)updateBackgroundAppearance {
-    if (self.transparent) {
-        [self.getAppearance configureWithTransparentBackground];
-        [self.getScrollEdgeAppearance configureWithTransparentBackground];
-    } else if (self.backgroundColor) {
-        [self.getAppearance setBackgroundColor:self.backgroundColor];
-        [self.getScrollEdgeAppearance setBackgroundColor:self.backgroundColor];
-    } else if (self.translucent) {
-        [self.getAppearance configureWithDefaultBackground];
-        [self.getScrollEdgeAppearance configureWithDefaultBackground];
-    } else {
+    if (@available(iOS 13.0, *)) {
         [self.getAppearance configureWithOpaqueBackground];
         [self.getScrollEdgeAppearance configureWithOpaqueBackground];
+
+        if (self.transparent) {
+            [self.getAppearance configureWithTransparentBackground];
+            [self.getScrollEdgeAppearance configureWithTransparentBackground];
+        } else if (self.backgroundColor) {
+            [self.getAppearance setBackgroundColor:self.backgroundColor];
+            [self.getScrollEdgeAppearance setBackgroundColor:self.backgroundColor];
+        } else if (self.translucent) {
+            [self.getAppearance configureWithDefaultBackground];
+            [self.getScrollEdgeAppearance configureWithDefaultBackground];
+        }
     }
```

Comment:
I think it make more sense to apply `configureWithOpaqueBackground` only when self.backgroundColor is available. We also do need to do the iOS 13 check because this class will load on iOS 13 and above anyways. See here: https://github.com/wix/react-native-navigation/blob/master/lib/ios/TopBarPresenterCreator.m

```suggestion
    if (self.transparent) {
        [self.getAppearance configureWithTransparentBackground];
        [self.getScrollEdgeAppearance configureWithTransparentBackground];
    } else if (self.backgroundColor) {
        [self.getAppearance configureWithOpaqueBackground];
        [self.getScrollEdgeAppearance configureWithOpaqueBackground];
        [self.getAppearance setBackgroundColor:self.backgroundColor];
        [self.getScrollEdgeAppearance setBackgroundColor:self.backgroundColor];
    } else if (self.translucent) {
        [self.getAppearance configureWithDefaultBackground];
        [self.getScrollEdgeAppearance configureWithDefaultBackground];
    }
```